pkg/registry/bundle.go

```diff
@@ -205,7 +206,13 @@ func (b *Bundle) cache() error {
 	for _, o := range b.Objects {
 		if o.GetObjectKind().GroupVersionKind().Kind == "CustomResourceDefinition" {
 			crd := &apiextensions.CustomResourceDefinition{}
-			if err := runtime.DefaultUnstructuredConverter.FromUnstructured(o.UnstructuredContent(), crd); err != nil {
+			// Marshal Unstructured and Unmarshal as CustomResourceDefinition. FromUnstructured has issues
```

Comment:
Can we file a bug upstream to document this for other devs?

Reply:
@alecmerdler yeah, this fix works for now but I need to nail down the exact wording. Maybe there's something additional we're doing wrong that's causing `FromUnstructured` to not work.